src/payment/strategies/paypal-commerce/paypal-commerce-hosted-form.ts

```diff
@@ -14,55 +11,44 @@ enum PaypalCommerceHostedFormType {
 export default class PaypalCommerceHostedForm {
     private _formOptions?: PaypalCommerceFormOptions;
     private _cardNameField?: PaypalCommerceRegularField;
-    private _hostedFields?: PaypalCommerceHostedFields;
     private _type?: PaypalCommerceHostedFormType;
 
     constructor(
-        private _paypalCommerceRequestSender: PaypalCommerceRequestSender
+        private _paypalCommercePaymentProcessor: PaypalCommercePaymentProcessor
     ) {}
 
-    async initialize(options: PaypalCommerceFormOptions, cartId: string, paypal: PaypalCommerceSDK) {
-        if (!paypal.HostedFields) {
-            throw new PaymentMethodClientUnavailableError();
-        }
+    async initialize(options: PaypalCommerceFormOptions, cartId: string, paramsScript: DataPaypalCommerceScript) {
```

Comment:
Is there a reason why changes to PayPal hosted credit card fields are bundled as a part of the PR for PayPal flow? It'd be nice to keep them as separate PRs unless they really have to go together.

Reply:
Actually, now feature with hosted fields doesn't work for shoppers (we're testing it yet)